source/more_sampling_tools.Rmd

```diff
@@ -26,15 +26,14 @@ source("_common.R")
 
 In the next chapter we will deal with some more involved problems in
 *probability*, as a preparation for *statistics*, where we use reasoning from
-probability to draw conclusions in a world like our own, where there appears to
-be random variation.
-
-Before we get down to the business of complex probabilistic problems in the
-next few chapters, let's consider a couple of peculiar puzzles that have uses
-for us here.  The first is, that they allow us to introduce some more of the
-key tools in {{< var lang >}} that we will use for Monte Carlo resampling. The
-second is that these problems are extremely valuable in showing the power of
-the Monte Carlo simulation method to help us solve, and then reason above,
+probability to draw conclusions about a world like our own, where variation
+often have the appearance of being random.
```

Comment:
```suggestion
often appears to be random.
```

Reply:
What I wanted to convey here is that the world is not typically random. But it has that appearance when enough factors come into play.

Reply:
"Appears to be random" suggests to me that it is not random. Or the opposite. Hard to know what it means!

Reply:
But I meant - doesn't "appears to be random" mean the same as "have the appearance of being random".   My reading of it, either way, is that it looks random, but may not be truly random.  Isn't that good enough?  Put more fully, we could say "we can make reasonable preliminary model of the variation as being random" - but that seems a bit too pregnant with content for the sentence there.

Reply:
The how about "acts (although isn't!) random".

Because it you tell me "X appears to be blind", it suggests to me that they are.

Reply:
How about 'appears to be more or less random'?